feat(changelog): cockpit-added-custom-retention-for-cockpits-datasources 2025-01-09 (#4192)
scaleway/docs-content: changelog/january2025/2025-01-09-cockpit-added-custom-retention-for-cockpits-datasources.mdx

---
title: Custom retention is now available for Cockpit data sources
status: added
author:
    fullname: 'Join the #observability channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2025-01-09
category: observability
product: cockpit
---

> Since January 1st 2025, you can [edit the retention duration](/observability/cockpit/how-to/change-data-retention/) of all your logs, metrics and traces data sources, and apply a different retention period to each of them depending on your needs. Extended retention was previously managed by Cockpit's premium plans which are now deprecated. Increasing your data retention above the default values is free of charge **during the beta phase only**, and will be charged again in a couple of months.  

Reviewer: ```suggestion
Since January 1st 2025, you can [edit the retention duration](/observability/cockpit/how-to/change-data-retention/) of all your logs, metrics and traces data sources, and apply a different retention period to each of them depending on your needs. Extended retention was previously managed by Cockpit's premium plans which are now deprecated. Increasing your data retention above the default values is free of charge **during the beta phase only**, and will become billable again in a couple of months.  
```

---

> title: Custom retention for Cockpit's datasources (beta)

Reviewer 2: ```suggestion
title: Custom retention is now available for Cockpit data sources
```

status: added
author:

>     fullname: ''

Reviewer 3: ```suggestion
    fullname: 'Join the #observability channel on Slack.'
```

>     fullname: ''

Reviewer 2: ```suggestion
    fullname: 'Join the #observability channel on Slack'
```

    url: 'https://slack.scaleway.com'
date: 2025-01-09
category: observability
product: cockpit
---

> Since January 1st 2025, you can edit the retention duration of all your logs, metrics and traces datasources, and apply a different retention period to each of them depending on your needs. Extended retention was previously managed by Cockpit's premium plans which are now deprecated. Increasing your data retention above the default values becomes free of charge during the beta phase *only*, and will be charged again in a couple of months.  
> 

Reviewer 2: ```suggestion
Since January 1st 2025, you can [edit the retention duration](/observability/cockpit/how-to/change-data-retention/) of all your logs, metrics and traces data sources, and apply a different retention period to each of them depending on your needs. Extended retention was previously managed by Cockpit's premium plans which are now deprecated. Increasing your data retention above the default values is free of charge **during the beta phase only**, and will be charged again in a couple of months.  
```
